specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2023-02-01/managedHsm.json

```diff
@@ -911,7 +911,8 @@
           "description": "SKU of the managed HSM Pool",
           "enum": [
             "Standard_B1",
-            "Custom_B32"
+            "Custom_B32",
+            "Custom_B6"
           ],
           "x-ms-enum": {
             "name": "ManagedHsmSkuName",
```

Comment:
Since you are doing breaking change review, please also check if they'd prefer you to change "modelAsString" to "true", especially if you are going to add new value to this in future.

Reply:
Also usually adding enum should happen in a new api-version.

Reply:
Recommendation from breaking change review board was keep it as is and not add 'modelAsString'.